stable/prometheus-mysql-exporter/README.md

```diff
@@ -57,12 +63,11 @@ The following table lists the configurable parameters of the mysql exporter char
 | `cloudsqlproxy.port`                     | Cloud SQL Proxy listening port                                                                                       | `3306`                                  |
 | `cloudsqlproxy.credentials`              | Cloud SQL Proxy service account credentials                                                                          | `bogus credential file`                 |
 
-
 Specify each parameter using the `--set key=value[,key=value]` argument to `helm install`. For example,
 
 ```console
 $ helm install --name my-release \
-  --set datasource="username:password@(db:3306)/"  \
+  --set mysql.user="username",mysq.password="password",mysql.host="localhost",mysql.port="3306"  \
```

Comment:
There is a "l" missing here in mysq.password

Reply:
Fixed :-)